pkgs/development/python-modules/mpv-jsonipc/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests
+, tqdm, websocket_client, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "mpv-jsonipc";
+  version = "unstable-2020-02-19";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "iwalton3";
+    repo = "python-mpv-jsonipc";
+    rev = "36eedbc0418211160a0b7fe3f0cc34db68ce4421";
+    sha256 = "08bs6qrcf5fi72jijmr2w7zs6aa5976dwv04f11ajwhj6i8kfq35";
+  };
+
+  # 'mpv-jsonipc' does not have any tests
+  doCheck = false;
+
+  propagatedBuildInputs = [ requests tqdm websocket_client ];
+
```

```suggestion

  pythonImportsCheck = [ "python-mpv-jsonipc" ];
```

give at least some assurance it works

---

Hm. It doesn't work. I'll look into this. Confusing since the application that uses this lib does still work... It didn't work with `python_mpv_jsonrpc` either... https://github.com/iwalton3/python-mpv-jsonipc/blob/master/setup.py

---

I guess, it's actually `python_mpv_jsonipc` despite what the setup.py says?